assets/views/layout.ejs

```diff
@@ -62,7 +62,7 @@
     </div>
 
     <!--SCRIPTS-->
-    <script src="/js/app.js"></script>
+    <script id="app" src="/js/app.js" _csrf="<%= _csrf %>"></script>
```

Comment:
I don't think ng-init will work, because we want the service function to be created before the controllers are created, but ng-init gets called after the controllers are created.